_component_design/chips.md

```diff
@@ -9,28 +9,28 @@ preview-image: preview-images/chip.svg
 resource: true
 status: stable
 last-modified: 2017-11-16
-helix-ui-css: false
-helix-ui-javascript: false
```

Comment:
Why were these bits of YAML removed? While not having them or setting them to false renders the same content onto the status page, it is foreseeable that the data be used differently in the future. I would prefer we not remove portions of the YAML.

For context as to what content these bits render see the status page or the following screenshot.

![image](https://user-images.githubusercontent.com/586682/36862216-c39a8e0c-1d4b-11e8-8e30-be4431bb9f98.png)

Marking these are `true` causes the checkmark to be rendered on this page.

Reply:
I honestly have no idea why that code was removed. I did not intentionally do that - hmmm...4 patterns were affected, I will add that code back in for each one.